pkg/services/services_test.go

```diff
@@ -0,0 +1,54 @@
+package services
+
+import (
+	"encoding/base64"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+)
+
+type ExampleAuthenticationTokenService struct{}
+type ExampleSignedToken struct {
+	rawAuthToken string
+}
+
+var _ AuthenticationTokenService = (*ExampleAuthenticationTokenService)(nil)
```

Comment:
why the unnamed nilpointer?

Reply:
Compiler time checking if the struct implements the interface.